File: module-chat/src/main/java/com/yyxnb/module_chat/bean/MessageBean.java
package com.yyxnb.module_chat.bean;

import com.yyxnb.what.core.interfaces.IData;

public class MessageBean implements IData<Long> {
    public int id;
    public int type;
    public String text;
    public String avatar;
    public String name;

    @Override
    public int id() {
        return id;
    }

    @Override
    public String getCode() {
        return null;
    }

    @Override
    public String getMsg() {
        return null;
    }

    @Override
    public Long getResult() {
        return null;
    }

    @Override
    public boolean isSuccess() {
        return false;
    }
}

File: module_chat/src/main/java/com/yyxnb/module_chat/bean/MessageBean.kt
package com.yyxnb.module_chat.bean

import com.yyxnb.common.interfaces.IData

data class MessageBean(
        var id: Int = 0,
        var type: Int = 0,
        @JvmField
        var text: String? = null,
        var avatar: String? = null,
        @JvmField
        var name: String? = null

) : IData<Long> {


    override fun id(): Int {
        return id
    }

    override fun getCode(): String {
        return id.toString()
    }

    override fun getMsg(): String? {
        return null
    }

    override fun getResult(): Long? {
        return null
    }

    override fun isSuccess(): Boolean {
        return false
    }
}